### PR TITLE
Added a new translation - Spanish

### PR DESCRIPTION
**Pull request by Jacxk**
This is now the one :D

### CODE REVIEW

File: src/de/False/BuildersWand/ConfigurationFiles/Locales.java [hunk 1]
```diff
@@ -111,8 +111,14 @@ private void setDefaults()
         de_de.put("reload", "Die Konfig wurde neugeladen.");
         de_de.put("playerNotFound", "Der Spieler konnte nicht gefunden werden.");
         de_de.put("noPermissions", "Du besitzt zu wenige Rechte um dies zu tun.");
+        
+		Map<String, String> es_es = new HashMap<>();
```
Comment (Falseee) [inline src/de/False/BuildersWand/ConfigurationFiles/Locales.java]: Pls remove the unnecessary whitespace in front of the map.

Comment (Jacxk) [reply src/de/False/BuildersWand/ConfigurationFiles/Locales.java]: sorry boss :(
